index.html

```diff
@@ -174,7 +174,7 @@ <h2 id="track-name"></h2>
         <h3>Track Search Results</h3>
         <div class="search-tracks" id="track-search-result">
           <ol>
-            <li></li>
+            <!-- <li></li> -->
```

Comment:
```suggestion
```

Reply:
deleted the <li> and added id on <h1> modal in track result page